docs/docsite/rst/roadmap/COLLECTIONS_4.rst

```diff
@@ -0,0 +1,49 @@
+===================
+Ansible project 4.0
+===================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-core <https://pypi.org/project/ansible-core/>`_ package as well. All dates are subject to change. See :ref:`base_roadmap_2_11` for the most recent updates on ``ansible-core``.
+
+.. contents::
+   :local:
+
+
+Release schedule
+=================
+
+
+:2021-01-26: New Collections will be reviewed for inclusion in ansible-4.
+:2021-03-03: ansible-4.0.0 alpha1 (biweekly ``ansible`` alphas.  These are timed to coincide with the start of the ``ansible-core-2.11`` pre-releases)
+:2021-03-17: ansible-4.0.0 alpha2
+:2021-03-30: ansible-4.0.0 alpha3
+:2021-04-13: last day for new collections to be submitted for inclusion in ansible-4. Note that collections MUST be reviewed and approved before being included. There is no guarantee that we will review every collection. The earlier your collection is submitted, the more likely it will be that your collection will be reviewed and the necessary feedback can be addressed in time for inclusion.
+:2021-04-13: ansible-4.0.0 alpha4
+:2021-04-14: Community IRC Meeting topic: list any new collection reviews which block release.  list any backwards incompatible collection releases that beta1 should try to accommodate.
+:2021-04-21: Community IRC Meeting topic: Decide what contingencies to activate for any blockers that do not meet the deadline.
+:2021-04-26: last day for new collections to be **reviewed and approved** for inclusion in ansible-4
+:2021-04-26: last day for collections to make backwards incompatible releases that will be accepted into Ansible-4.
+:2021-04-27: ansible-4.0.0 beta1 -- feature freeze [1]_ (weekly beta releases.  Collection owners and interested users
+             should test for bugs)
```

Comment:
extra whitespace?

```suggestion
:2021-04-27: ansible-4.0.0 beta1 -- feature freeze [1]_ (weekly beta releases.  Collection owners and interested users should test for bugs)
```

Reply:
I cut lines at 100 characters for easy reading as text.  But rst will parse either one correctly when transforming to html.